src/Raven.Server/Dashboard/ThreadsInfoNotifications.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Threading;
+using Raven.Server.NotificationCenter;
+using Raven.Server.ServerWide;
+
+namespace Raven.Server.Dashboard;
+
+public class ThreadsInfoNotifications : NotificationsBase
+{
+    public ThreadsInfoNotifications(CancellationToken shutdown)
+    {
+        var options = new ThreadsInfoOptions();
+
+        var threadsInfoNotificationSender = new ThreadsInfoNotificationSender(nameof(ServerStore), Watchers, options.ThreadsInfoThrottle, shutdown);
+        BackgroundWorkers.Add(threadsInfoNotificationSender);
```

Comment:
Do we want that? Shouldn't we only monitor threads when view is opened?

Reply:
It isn't actually monitored until we have someone who connects to see this info.

Reply:
@ArieSLV can you confirm?